CRM/Fbpixel/Upgrader/Base.php

```diff
@@ -8,39 +8,39 @@
 class CRM_Fbpixel_Upgrader_Base {
 
   /**
-   * @var varies, subclass of ttis
```

Comment:
Hi Lucky. What was the civilint error on this line? (If you change it back and run civilint again, you should see the error message.)

Reply:
The same as in the screenshots.
```
Expected "variessublcassofttis" but found "varies, subclass of ttis" for @var tag in member variable comment
```

Reply:
Okay, thanks @lucky091588 . I think we'd better ignore this error message and leave the lines as they are, e.g.,
```
   * @var varies, subclass of ttis
```
I'm not sure why civilint is reporting this as an error; I'll try to ask the civilint developer about it. But for now, it just seems unnecessary to turn a normal English comment into a one-word string, just to pass this lint test. Thanks!